SFFeatureFlag/Classes/Services/FeatureFlagCenter.swift

```diff
@@ -0,0 +1,159 @@
+//
+//  FeatureFlagCenter.swift
+//  SFFeatureFlag
+//
+//  Created by Maja Sapunova on 8.6.21.
+//
+
+import Foundation
+
+public class FeatureFlagCenter {
+    
+    enum FeaturesConfigState {
+        case needsRefresh
+        case loading
+        case loaded
+    }
+    
+    //MARK: - Properties
+    
+    /// The key that will be used for getting the features config data from the remote config service. By default is set to 'featues_config'
+    public var featuresConfigKey = "features_config"
+    
+    /// The decoding strategy that will be used when decoding the features config data
+    public var featuresConfigDecodingStrategy = JSONDecoder.KeyDecodingStrategy.convertFromSnakeCase
+    
+    private let remoteConfigService: RemoteConfigServiceProtocol
+    private var featuresConfigState: FeaturesConfigState = .needsRefresh
+    
+    private lazy var featuresConfig: [FeatureConfig] = {
+        return getFeaturesConfig()
+    }()
+    
+    //MARK: - Private Methods
```

Comment:
The description of the MARK is probably mistaken. I don't see any Private methods below it.